src/main/java/edu/kis/vh/nursery/ownStackImplementations/IntArrayStack.java

```diff
@@ -23,14 +22,14 @@ public boolean isFull() {
     @Override
     public int pop() {
         if (isFull())
-            return EMPTY_RETURN_CODE;
+            return EMPTY_RETURN_CODE + 1;
```

Comment:
It should be enough to change a value of the constant, otherwise, it does not make sense.